### DISCUSSION TITLE
Beginner Question : How can I deploy Datasette to Digital Ocean droplet instead of Heroku?

### DISCUSSION THREAD

**Discussion by GmGniap** [simonw/datasette#1381] [Q&A] [answered]
Hi all , May I ask a newbie question regarding to deployment? 
I'm totally beginner to that server field and now using Datasette with Heroku deployment. But, when my database is growing up , I would like to move to Digital Ocean to get better performance. But I can't find any reference or tutorial on Internet. 

So, if possible , could you please suggest me how should I continue - workflow or process ideas to keep me in right way? 
Thanks & Regards ,

**Comment by simonw** [comment, marked as answer]
The new DigitalOcean App Platform supports Heroku-style buildpacks - I've deployed Datasette to that successfully before. Some details on how to do that are here: https://docs.datasette.io/en/stable/deploying.html#deploying-using-buildpacks

If you want to use a DigitalOcean droplet I recommend using Ubuntu and systemd, see https://docs.datasette.io/en/stable/deploying.html#running-datasette-using-systemd
